src/app/(BottomTabNavigation)/AllCases/CaseScreen/[caseUid].tsx

```diff
@@ -25,6 +27,21 @@ function CaseScreen() {
     setIsLoading(false);
   };
 
+  const onShare = async () => {
+    const addCaseUrl = Linking.createURL('?event=addCase&', {
+      queryParams: { caseUid },
+    });
```

Comment:
`event=addCase` is technically a query parameter, so maybe we can clean up this function instead of concatenating multiple partial strings.